src/com/amplitude/api/AmplitudeClient.java

```diff
@@ -194,6 +194,7 @@ public AmplitudeClient() {
      */
     public AmplitudeClient(String instance) {
         this.instanceName = Utils.normalizeInstanceName(instance);
+        this.url = Constants.EVENT_LOG_URL;
```

Comment:
I kinda like leaving this set when the variable is declared, but either way is fine.

Reply:
oh that actually makes sense and looks cleaner